generate_updateinfo.py

```diff
@@ -312,13 +317,16 @@ def build_updateinfo(src):
                 rel_fd[p_release].write('  <update from="%s" status="stable" type="bugfix" version="1.4">\n' % UPDATE_FROM)
             if sec_dict._attrs['type'] == adv_types['enhancement']:
                 rel_fd[p_release].write('  <update from="%s" status="stable" type="enhancement" version="1.4">\n' % UPDATE_FROM)
-            rel_fd[p_release].write("    <id>%s</id>\n" % i)
+            rel_fd[p_release].write("    <id>%s</id>\n" % pretty_i)
             rel_fd[p_release].write("    <title>%s</title>\n" % sec_dict._attrs['synopsis'])
             rel_fd[p_release].write("    <release>CentOS %s</release>\n" % p_release)
             rel_fd[p_release].write("    <issued date=\"%s\" />\n" % sec_dict._attrs['issue_date'])
             if sec_dict._attrs['type'] == adv_types['security']:
                 rel_fd[p_release].write("    <severity>%s</severity>\n" % sec_dict._attrs['severity'])
             rel_fd[p_release].write("    <references>\n")
+            if 'keywords' in sec_dict._attrs:
+                if re.search("reboot_suggested", sec_dict._attrs['keywords']):
+                    rel_fd[p_release].write("    <reboot_suggested>1</reboot_suggested>\n")
```

Comment:
The reboot_suggested tag should not be inside the references tag.

Can you adjust your pull-request to move it?

Reply:
I honestly don't know how.
I have neither enough Python nor enough Github skills :-/

Reply:
line 321 needs to be moved below the reboot_suggested lines